settings.gradle

```diff
@@ -63,6 +63,9 @@ project(':core').projectDir = new File(settingsDir, '../../android-sdk/sdk/core/
 include ':DownloadToOwnSampleApp'
 project(':DownloadToOwnSampleApp').projectDir = new File(settingsDir, './PlaybackLab/DownloadToOwnSampleApp/app')
 
+include ':ClearExoPlayerSampleApp'
```

Comment:
I don't think we need to include this project as part of our all apps project. It's not really a project we want to have for a long time, it's just to verify against our solution.

Reply:
Agree with @rcvrgs  here. 
I think we need to see how Exo 2.6.1 downloader behaves on different situations which we are observing on our player. Like in case of 4.4 devices, also can we support different bitrates etc